src/control/server/drpc.go

```diff
@@ -66,12 +67,14 @@ func checkSocketDir(sockDir string) error {
 }
 
 type drpcServerSetupReq struct {
-	log     logging.Logger
-	sockDir string
-	engines []Engine
-	tc      *security.TransportConfig
-	sysdb   *raft.Database
-	events  *events.PubSub
+	log        logging.Logger
+	sockDir    string
+	engines    []Engine
+	tc         *security.TransportConfig
```

Comment:
nit: not relevant for this PR but we should really change tc to transport

Reply:
Renamed to `transportConfig`.